traffic/traffic_info.cpp

```diff
@@ -22,18 +22,25 @@ namespace traffic
 {
 namespace
 {
-bool ReadRemoteFile(string const & url, vector<uint8_t> & result)
+bool ReadRemoteFile(string const & url, string & result, int & errorCode)
```

Comment:
Could you please get back to vector<uint8_t> here? I suggest you to convert this vector to string (which seems quite fast as this is the case for small vectors) when you need to retrieve version in case of error.